modules.yml

```diff
@@ -1020,3 +1020,13 @@
   path: recon/netblocks-hosts/reverse_resolve
   required_keys: []
   version: '1.0'
+- author: Ryan Hays
+  dependencies: []
+  description: Uses the BinaryEdge API to discover subdomains.
+  files: []
+  last_updated: '2019-05-28'
+  name: BinaryEdge.io DNS lookup
+  path: dev/binaryedge_hosts
+  required_keys:
+  - binaryedge_api
+  version: '1.0'
```

Comment:
Always add line break to the end of the file. Thanks.

Reply:
Try now